src/components/VTooltip/VTooltip.js

```diff
@@ -130,6 +130,15 @@ export default {
     }
   },
 
+  watch: {
+    positionX () {
+      this.updateDimensions()
```

Comment:
Maybe it can be simplified to `positionX: this.updateDimensions` or `positionX: 'updateDimensions'`

Reply:
```suggestion
```